test/RateTargetBaseInterestRateStrategy.t.sol

```diff
@@ -57,12 +57,28 @@ contract RateTargetBaseInterestRateStrategyTest is InterestRateStrategyBaseTest
         assertEq(interestStrategy.getBaseVariableBorrowRate(), 0.055e27);
         assertEq(interestStrategy.getMaxVariableBorrowRate(),  0.515e27);
 
-        rateSource.setRate(0.07e27);
+        rateSource.setRate(0.07e18);
+        rateSource.setDecimals(18);
```

Comment:
Lets move this to a dedicated test, testing two things at once here - also can update test names to be more specific `changeRate` `changeDecimals` or something

Reply:
done